modular_zubbers/code/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -1,3 +1,249 @@
+/datum/job/ai/New()
+	.=..()
+	alt_titles += list(
+		"Abominable Intelligence",
+		"Artificial Intelligence",
+		"Big Brother",
+		"Big Sibling",
+		"Big Sister",
+		"Machine Spirit",
+	)
+
+/datum/job/assistant/New()
+	.=..()
+	alt_titles += list(
+		"Bastard",
+		"Call Boy",
+		"Call Girl",
+		"Callperson",
+		"Crew Gremlin",
+		"Escort",
+		"Fiend",
+		"Fleshlight",
+		"Hooker",
+		"Jackanape",
+		"Maintworker",
+		"Ne'er-do-well",
+		"Prostitute",
+		"Roustabout",
+		"Sexworker",
+		"Slut",
+		"Spacefarer",
+		"Stripper",
+		"Troublemaker",
+		"Whore",
```

Comment:
**no.** no. absolutely not.

Reply:
I took out the ones not referring to a job, hopefully that works